genestack_client/file_filters.py

```diff
@@ -185,3 +185,21 @@ class OrFileFilter(FileFilter):
     def __init__(self, first, second):
         super(OrFileFilter, self).__init__()
         self._dict.update({'or': [first.get_dict(), second.get_dict()]})
+
+
+class MetainfoRelatedValueFilter(FileFilter):
+    """
+    Filter to select files with metainfo values related to given terms by a dictionary relationship.
+    """
+    def __init__(self, key, term_labels, dictionary_accession, relationship_label, transitive):
```

Comment:
Can you add docstrings about arguments. 

Some examples of the `relationship_label` and description of what transitive do.

Reply:
Done